src/include/platform/ConfigurationManager.h

```diff
@@ -328,6 +330,13 @@ inline CHIP_ERROR ConfigurationManager::GetPairedAccountId(char * buf, size_t bu
     return static_cast<ImplClass *>(this)->_GetPairedAccountId(buf, bufSize, accountIdLen);
 }
 
+#if CHIP_ENABLE_ROTATING_DEVICE_ID
+inline CHIP_ERROR ConfigurationManager::GetLifetimeCounter(uint16_t & rotationCouter)
```

Comment:
i used rotaton-counter terminology at first, but @tcarmelveilleux had a good point of using lifetime-counter since it's not going to be rotated that's why from a implementation perspective (especially other component can use it) it's better to rename it to lifetime-counter.

Reply:
OK, but what about documenting what this thing is, then?  We're adding an API to ConfigurationManager.h whose behavior is completely unclear, since it has neither documentation nor spec basis.

Reply:
i agree, i added some documentation about the counter and when it gets incremented